homeassistant/components/plex/config_flow.py

```diff
@@ -49,9 +49,7 @@
     DOMAIN,
     MANUAL_SETUP_STRING,
     PLEX_SERVER_CONFIG,
-    X_PLEX_DEVICE_NAME,
-    X_PLEX_PLATFORM,
-    X_PLEX_PRODUCT,
+    X_PLEX_ID_STRING,
```

Comment:
This is okay, but in my opinion not ideal. Each of X_PLEX_DEVICE_NAME, X_PLEX_PLATFORM, and X_PLEX_PRODUCT are still a different variable used in different places (and could change in the future). It is just that they happen to equal the same string at this time. Therefore it would be better to have

X_PLEX_DEVICE_NAME = "Home Assistant"
X_PLEX_PLATFORM = X_PLEX_DEVICE_NAME
X_PLEX_PRODUCT = X_PLEX_DEVICE_NAME

and keep how they use the variables the same.
Doesn't *need* change but if you do change something else you can change this

Reply:
I agree that your solution is probably better and cleaner. However, since the other solution is already implemented, works, and has not changed functionality, I'll keep it :)